app/models/concerns/wakes/metrics/facebook.rb

```diff
@@ -7,13 +7,26 @@ module Facebook
       included do
         store_accessor :document, :facebook_count, :facebook_count_updated_at
 
+        before_validation :prevent_update_of_legacy_location, on: :update
```

Comment:
@benhutton The issue then is that `facebook_count_updated_at` will be updated even though `facebook_count` is not: https://github.com/desiringgod/wakes/blob/develop/app/services/wakes/facebook_count_updater_service.rb#L14. 

While we could code around that, that smells like we want the transaction to fail, not the individual attribute (especially since we can't predict how future develops will try to set the attribute). Doesn't validating on save feel more aligned with ActiveModel, too?